src/development/platform-integration/ios/apple-frameworks.md

```diff
@@ -0,0 +1,200 @@
+the package in your Dart file. 
+
+Occasionally, you may also need to change app settings or 
+initialization logic- which will be explained in the package's 
+"Readme" page on [pub.dev][].
+
+### Accessing the photo library
+In Swift and Objective-C, you use `PhotoKit` and the `PhotosUI` framework
+to access user photos.
+
+In Flutter, you can use the [`image_picker`][] plugin, 
+which supplies an `ImpagePicker` class that is used to 
+display the user's photo library.
+
+### Accessing the camera
+In SwiftUI and UIKit, to allow your app's users to take a photo, 
+you use the `UIImagePickerController` class, 
+with the `.camera` `sourceType`.
+
+In Flutter, to access the camera, 
+you'll use the same [`image_picker`][] plugin that was 
+referenced above. The code will look similar, 
+only now the `source` for `pickImage` is the camera.
+
+### In-app purchases
+In Swift and Objective-c, you use `StoreKit` to 
+offer in-app purchases.
+
+In Flutter, you can support in-app purchases through the 
+Google Play store on Android and the 
+Apple App Store on iOS by using the 
+[`in_app_purchase`][] plugin.
+
+### Payment processing
+In Swift and Objective-C, you use `PassKit` to process 
+Apple Pay payments in your app. 
+
+In Flutter, you can use the [`pay`][] plugin to 
+add Google Pay payments on Android
+ and Apple pay payments on iOS.
+
+### Push notifications
+In Swift and Objective-C, you use the `UserNotifications` 
+framework for pushing user-facing notifications.
+
+In Flutter, you can use the [`firebase_messaging`][] plugin, 
+which uses Firebase Cloud Messaging. 
+
+### Accessing GPS coordinates
+In Swift and Objective-C, you use the `CLLocationManager` class, 
+of the `CoreLocation` framework, 
+to access the user's location. 
+
+In Flutter, to access the user's location, 
+you can use the [`geolocator`][] plugin. 
+
+### Accessing sensor data
+In Swift and Objective-C, you use the `CMMotionManager` class,
+of the `CoreMotion` framework, to access sensor data
+like the accelerometer or the gyroscope.
+
+In Flutter, to read sensor data, 
+you can use the [`sensors_plus`][] plugin.
+
+### Embedding maps
+In Swift and Objective-c, you use `MapKit` to embed 
+Apple Maps within your app. 
+
+In Flutter, you can embed Google Maps using the 
+[`google_maps_flutter`][] plugin. 
+
+### Network requests
+In Swift and Objective-C, to make an HTTP request you use the
+ `URLSession` class.
+
+In Flutter, you can use the [`http`][] package.
+
+### Persisting data
+In SwiftUI, to persist small pieces of data in your application, 
+you would likely use the `@AppStorage` property wrapper.
+
+In Flutter, to utilize the operating system's key-value storage, 
+you can use the [`shared_preferences`][] plugin. 
+
+### Accessing health data
+In Swift and Objective-C, you use the `HealthKit` framework
+to access health and fitness data.
+
+In Flutter, you can use the [`health`][] plugin 
+to access `HealthKit` data.
```

Comment:
this seems 3P plugin too. should we be careful about endorsing a 3P plugin in our official doc?

Reply:
Yes, correct, we need to be careful.

Reply:
There are numerous 3P plugins in this doc. Is there a way we can mention them without "endorsing"? Maybe a sentence at the beginning saying we are not endorsing?

Reply:
iirc there's something called 'flutter favorite program'? maybe only list things that belong to that?

Reply:
Yes but it hasn't run in a few years so newer packages aren't on there, I'm working on that separately but I think that would be a blocker for now